doc/modules/language-guide/examples/grammar.txt

```diff
@@ -169,6 +169,8 @@
     'actor' <exp_plain>
     'not' <exp_un>
     'debug_show' <exp_un>
+    TO_C'and'<id> '(' <list(<exp>, ',')> ')'
+    FROM_C'and'<id> <exp_un>
```

Comment:
To fix this, you'll need to adjust the grammar massaging sed script in motoko/src/grammar.sed to replace TO_CANDID and FROM_CANDID by lc versions, probably before the rewrite of AND to avoid interference.

Reply:
Finally got to this.

I followed your advice, but also needed to hoist above 'ID' as well as 'AND'.